pep/client.go

```diff
@@ -0,0 +1,87 @@
+package pep
+
+//go:generate bash -c "mkdir -p $GOPATH/src/github.com/infobloxopen/policy-box/pdp-service && protoc -I $GOPATH/src/github.com/infobloxopen/policy-box/proto/ $GOPATH/src/github.com/infobloxopen/policy-box/proto/service.proto --go_out=plugins=grpc:$GOPATH/src/github.com/infobloxopen/policy-box/pdp-service && ls $GOPATH/src/github.com/infobloxopen/policy-box/pdp-service"
+
+import (
+	"fmt"
+	"reflect"
+	"time"
+
+	"golang.org/x/net/context"
+	"google.golang.org/grpc"
+
+	pb "github.com/infobloxopen/policy-box/pdp-service"
+)
+
+var (
+	ErrorConnected    = fmt.Errorf("Connection has been already established")
+	ErrorNotConnected = fmt.Errorf("No connection")
+)
+
+type Client struct {
+	addr   string
+	conn   *grpc.ClientConn
+	client *pb.PDPClient
+}
+
+func NewClient(addr string) *Client {
+	return &Client{addr: addr}
+}
+
+func (c *Client) Connect(timeout time.Duration) error {
+	if c.conn != nil {
+		return ErrorConnected
+	}
+
+	conn, err := grpc.Dial(c.addr, grpc.WithInsecure(), grpc.WithBlock(), grpc.WithTimeout(timeout))
```

Comment:
Fine for now but this will have to change to allow TLS. Also, the WithBalancer option https://github.com/grpc/grpc-go/blob/master/clientconn.go#L130 

Why the WithBlock? Seems unnecessary here.